docs/syntax-reference.rst

```diff
@@ -39,6 +39,32 @@ means new facts can be inserted to the table, and existing facts can be updated
 Kafka topic or derived from existing streams and tables. In both cases, a table’s underlying data is durably stored (persisted)
 within a Kafka topic on the Kafka brokers.
 
+.. _struct_overview:
+
+STRUCT
+------
+
+In KSQL 5.0 and higher, you can read nested data, in Avro and JSON formats,
+by using the STRUCT type in CREATE STREAM and CREATE TABLE statements. Use the
+following syntax to declare nested data: 
+
+.. code:: sql
+
+   STRUCT<FieldName FieldType, ...>
+
+The STRUCT type requires you to specify a list of fields. For each field, you
+specify the field name and field type. The field type can be any of the
+supported KSQL types, including the complex types ``MAP``, ``ARRAY``, and
+``STRUCT``.
+
+Access the fields in a ``STRUCT`` by using the struct dereference operator
+(``->``). For more info, see :ref:`operators`.
+
```

Comment:
I think we should add an example here.  One of my general complaints about the Syntax Guide is that we almost never show even a single example.  Instead, users have to use two browser tabs or jump back and forth between the Syntax Guide and the Examples pages.

Reply:
PS: I see an example further down below, but you need to be able to find that (when reading this section here).